x-pack/plugins/fleet/server/services/artifacts/artifacts.test.ts

```diff
@@ -186,13 +186,25 @@ describe('When using the artifacts services', () => {
     });
 
     it('should create and return a multiple big artifacts', async () => {
-      const { ...generatedArtifact1 } = generateArtifactMock({ encodedSize: 5_000_500 });
+      const { ...generatedArtifact1 } = generateArtifactMock({
```

Comment:
why do you need to spread rest like this?
```suggestion
      const generatedArtifact1 = generateArtifactMock({
```

Reply:
Good point, this code was already there, but what you pointed out makes totally sense

Reply:
@tomsonpl https://github.com/elastic/kibana/pull/160710/commits/0b6e726af6f3b00455627e414352404fc0dcc0a3